rviz_visual_testing_framework/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ option(EnableVisualTests "decides whether or not enable the tests")
 
 # Path variables needed to save and find screenshots for image comparison.
 add_definitions(-D_BUILD_DIR_PATH="${CMAKE_CURRENT_BINARY_DIR}")
-add_definitions(-D_SRC_DIR_PATH="${CMAKE_CURRENT_SOURCE_DIR}")
+#add_definitions(-D_SRC_DIR_PATH="${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Please remove lines rather than comment them.

Reply:
Noted. I will leave this uncommented since its probably out of scope of this PR and restrict changes to the test case in cpp